catkin_ws/src/bowser_2dnav/yaml/costmap_common_params.yaml

```diff
@@ -0,0 +1,38 @@
+obstacle_range: 5
+raytrace_range: 5
+footprint: [[0.45,-0.41], [0.45,0.41], [-0.45,0.41], [-0.45,-0.41]]
+footprint_padding: 0.03
+inflation_radius: 0.3
+transform_tolerance: 1
+max_obstacle_height: 9999.9999
+min_obstacle_height: .4
+
+observation_sources: point_cloud_obstacles point_cloud_ground
+
+# point_cloud_sensor1: {
```

Comment:
Can we get rid of this?

Reply:
This is what we call "inline documentation." If it isn't still there, I'll forget about it.